idd/Energy+.idd.in

```diff
@@ -34502,6 +34502,4630 @@ ZoneHVAC:EvaporativeCoolerUnit,
+	A10,	\field Outside Air Node Name
+		\type node
+		\note Outside air node for the hybrid unit must be an outdoor air node.
+	A11,	\field Supply Air Node Name
+		\type node
+		\note Supply air node for the hybrid unit must be a zone air inlet node.
+	A12,	\field Relief Node Name
+		\type node
+		\note Relief node for the hybrid unit must be a zone exhaust node, unless flow is being balanced elsewhere.
+	N1,	\field System Maximum Supply AirFlow Rate
+		\type real
+		\minimum> 0
+		\note The value in this field represents the maximum supply air volume flow rate among all operating modes. Values of extensive variables in lookup tables are normalized by the system maximum supply air mass flow rate that was used to build performance curves. The value in this field is used to rescale the output from exenstive variables to a desired system size.
+		\units m3/s
+	N2,	\field External Static Pressure at System Maximum Supply Air Flow Rate
+		\type real
+		\minimum> 0
+		\note Input the external static pressure when the system operates at maximum supply air flow rate. Fan affinity laws are used to scale supply fan power from the values tabulated in lookup tables, to values that match the external static pressure input to this field. If this field is blank, the supply fan power is not scaled from the values tabulated in lookup tables.
+		\units Pa
+	N3,	\field Scaling Factor
+		\type real
+		\minimum> 0
+		\note The value in this field scales all extensive performance variables including: supply air mass flow rate, fuel uses, and water use. If this field is blank, the default scaling factor is 1.
+		\default 1
+	N4,	\field Number of Operating Modes
+		\type integer
+		\minimum 1
+		\maximum 26
+		\note The value in this field defines the number of discrete operating modes for the unitary hybrid equipment. Supply air mass flow rate ratio and outside air fraction are treated as continuous controlled inputs within each discrete operating mode.
+		\note Mode0 always describes standby.
+		\note Mode1 - Mode 25 describe active operating modes that provide ventilation, heating, cooling, humidification or dehumidification.
+		\note 26 modes are allowed (including Mode0).
+		\note If the value input to this field is 1, only Mode0 will be considered.
+		\note If this field is blank, the default number of operating modes is 1.
+		\default 1
+	N5,	\field Minimum Time Between Mode Change
+		\type real
+		\minimum 1
+		\note Any mode selected will not operate for less time than the value input in this field. If the value in this field is larger than each timestep, the mode and settings selected in one time step will persist in later time steps until the minimum time between mode change is satisfied. Mode0 does not have a minimum time. If this field is blank, the default minimum time between mode change is 10 minutes.
+		\units minutes
+		\default 10
+	A13,	\field First fuel type
+		\type choice
+		\key Electricity
+		\default Electricity
```

Comment:
I've expanded the key choices. This way a hybrid system that does not consume electricity could be modeled.  (eg: gas engine driven heat pump)